core/src/avm1/globals/context_menu_item.rs

```diff
@@ -20,7 +20,10 @@ pub fn constructor<'gc>(
         .get(0)
         .unwrap_or(&Value::Undefined)
         .coerce_to_string(activation)?;
-    let callback = args.get(1).map(|v| v.coerce_to_object(activation));
+    let callback = args
+        .get(1)
+        .filter(|v| v.type_of() != Value::Undefined.type_of())
```

Comment:
to be sure, why can't we directly compare with `undefined` here? I haven't seen this pattern of comparing with `Undefined.type_of()` before.

Other than this, the PR looks fine.